scripts/docker_migrate.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+set -eE
+
+trap 'handler' ERR
+
+handler () {
+  echo
+  echo 'An error occurred running this migration. Your existing Firezone installation has not been affected.'
+  rm .env
+  exit 1
+}
+
+curlCheck () {
+  if ! type curl > /dev/null; then
+    echo 'curl not found. Please install curl to use this script.'
+    exit 1
+  fi
+}
+
+dockerCheck () {
+  if ! command -v docker > /dev/null; then
+    echo 'docker not found. Please install docker and try again.'
+    exit 1
+  fi
+}
+
+prompt () {
+  echo 'This script will copy Omnibus-based Firezone configuration to Docker-based Firezone configuration.'
+  echo 'It operates non-destructively and leaves your current Firezone services running.'
+  read -p 'Proceed? (Y/n): ' migrate
+
+  case $proceed in
+    n|N) echo 'Aborted' ;;
+    *) migrate
+  esac
+}
+
+condIns () {
+  dir=$1
+  file=$2
+
+  if [ -s "$dir/$file" ]; then
+    val=$(cat $dir/$file)
+    val=$(echo $val | sed 's/"/\\"/g')
+    if [ $file = "EXTERNAL_URL" ]; then
+      val=$(echo $val | sed 's:/*$::')
+    fi
+    echo "$file=\"$val\"" >> .env
+  fi
+}
+
+migrate () {
+  env_files=/opt/firezone/service/phoenix/env
+  cwd=`pwd`
+  read -p "Enter the desired installation directory ($cwd): " installDir
+  if [ -z "$installDir" ]; then
+    installDir=$cwd
+  fi
+  cd $installDir
+  if ! test -f docker-compose.yml; then
+    curl -fsSL https://raw.githubusercontent.com/firezone/firezone/master/docker-compose.prod.yml -o docker-compose.yml
```

Comment:
just a note to other reviewers, this is pointing to master, need to update it for testing :sob: maybe we can fix this to a tag before merging

Reply:
Hm. I think we probably still want to use `master` for fetching raw scripts.